src/libraries/System.Private.Xml/src/System/Xml/Dom/XmlNodeReader.cs

```diff
@@ -1327,7 +1339,7 @@ public override string GetAttribute(string name)
         }
 
         // Gets the value of the attribute with the specified name and namespace.
-        public override string GetAttribute(string name, string namespaceURI)
+        public override string? GetAttribute(string name, string namespaceURI)
```

Comment:
`namespaceURI` can be null

Reply:
yes, here and couple of other places in this file. Although some methods taking namespaceURI will currently throw NRE which I suspect we could potentially fix although I will leave it out of scope for this PR until someone complains since it's not a breaking change to relax nullability of arguments